pkg/deploy/deploy.go

```diff
@@ -107,6 +115,7 @@ func Deploy(ctx context.Context, projects []project.Project, environmentClients
 			log.WithFields(field.Environment(env.Name, env.Group)).Info("Deployment successful for environment %q", env.Name)
 		}
 	}
+	reporter.ReportInfo("Deployment finished")
```

Comment:
Line 112 is preventing me from seeing `Deployment finished` if there is an error. A hack would be to `defer` the call to `ReportInfo(...)`

Reply:
Added a defer on top of it https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1694/files#diff-39e0509b5e221f87189994b158c992774a7be2022357c00e9266f73c7f2f9c2eR95-R97